source/jobs.erb

```diff
@@ -162,7 +162,7 @@ published: true
 
   <%= partial 'jobs/quote_caricature', locals: { section: current_page.data.alumni_quote } %>
 
-  <%= partial 'jobs/contact_us' %>
+  <%= partial 'jobs/contact_us', locals: { roles: current_page.data.roles.content.length } %>
```

Comment:
We could perhaps use something like `roles_count` or similar for this name...I think `roles` could mean a collection.